extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -51,12 +51,13 @@ export class LocalSubscriptionPlatform {
    * @return {!Promise<!Entitlements>}
    */
   getEntitlements() {
-    user().assert(this.serviceConfig_['paywallUrl'],
-        'Service config does not have paywall Url');
+    const authUrl = user().assert(this.serviceConfig_['authorizationUrl'],
+        'Service config does not have authorization Url');
     const currentProductId = user().assertString(
         this.pageConfig_.getProductId(), 'Current Product ID is null');
+
     return this.xhr_
-        .fetchJson(this.serviceConfig_['paywallUrl'])
+        .fetchJson(authUrl)
```

Comment:
We need to make sure all CORS stuff is added here.

Reply:
Is there something like a util for this?

Reply:
It's the `fetchJson` API. I believe it's something like this:
```
fetchJson(url, {
  credentials: 'include',
})
```

But please double-check.

Reply:
Done! I guess!